resources/assets/js/components/partials/form/Textarea.vue

```diff
@@ -11,78 +11,61 @@ textarea:focus {
 
 <template>
   <div>
-    <p class="mb2" :class="{ b: required }" v-if="! noLabel">{{ label }}</p>
+    <p v-if="label != ''" class="mb2" :class="{ b: required }">
+      {{ label }}
+    </p>
     <textarea
-            v-model="buffer"
-            @input="$emit('contentChange', buffer)"
-            autofocus
-            :required="required"
-            :name="id"
-            :placeholder="placeholder"
-            :id="id"
-            :rows="rows"
-            class="br2 f5 w-100 ba b--black-40 pa2 outline-0" :style="'width:' + width + 'px'">{{ buffer }}</textarea>
+      :id="id"
+      v-model="buffer"
+      autofocus
+      :required="required"
+      :name="id"
+      :placeholder="placeholder"
+      :rows="rows"
+      class="br2 f5 w-100 ba b--black-40 pa2 outline-0"
+      :style="'width:' + width + 'px'" @input="$emit('contentChange', buffer)"
+    ></textarea>
```

Comment:
The ``{{buffer}}`` textarea content is not needed, because of the `v-model` placeholder